src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -513,6 +515,35 @@ private static string[] GetDefaultLetterKeyDE(LetterKey letter)
             };
         }
 
+        // Greek
+        private static string[] GetDefaultLetterKeyEL(LetterKey letter)
+        {
+            return letter switch
+            {
+                LetterKey.VK_A => new string[] { "α" },
+                LetterKey.VK_B => new string[] { "β" },
+                LetterKey.VK_C => new string[] { "χ" },
+                LetterKey.VK_D => new string[] { "δ" },
+                LetterKey.VK_E => new string[] { "ε", "η" },
+                LetterKey.VK_G => new string[] { "γ" },
+                LetterKey.VK_I => new string[] { "ι" },
+                LetterKey.VK_K => new string[] { "κ" },
+                LetterKey.VK_L => new string[] { "λ" },
+                LetterKey.VK_M => new string[] { "μ" },
+                LetterKey.VK_N => new string[] { "ν" },
+                LetterKey.VK_O => new string[] { "ο", "ω" },
+                LetterKey.VK_P => new string[] { "π", "φ", "ψ" },
+                LetterKey.VK_R => new string[] { "ρ" },
+                LetterKey.VK_S => new string[] { "σ" },
+                LetterKey.VK_T => new string[] { "τ" },
```

Comment:
```suggestion
                LetterKey.VK_T => new string[] { "τ", "θ" },
```
I think you forgot to add θ (theta)

Reply:
Added Theta to T and Phi is now part of F and P